testing/php7-yaml/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# vim: smartindent noexpandtab tabstop=4 shiftwidth=4
+# Maintainer: Renoir Boulanger <hello@renoirboulanger.com>
+
+pkgname=php7-yaml
+_pkgreal=yaml
+pkgver=2.0.0
+pkgrel=0
+_phpver=${pkgname#php}
+_phpver=${_phpver%%-*}
+pkgdesc="PHP${_phpver}"
+url="http://bd808.com/pecl-file_formats-yaml/"
+arch="all"
+license="MIT"
+makedepends="php${_phpver}-dev autoconf yaml-dev"
+source="${pkgname}-${pkgver}.tgz::http://pecl.php.net/get/${_pkgreal}-${pkgver}.tgz"
```

Comment:
What's the convention guideline,  is it written somewhere?

if I do `source="foo.tgz::http://example.org/a/b/foo.tgz"` it *will* write the file as `foo.tgz` (that is fine). So any name might fit there.  But we still gotta check what's the folder name when we extract (e.g. extracted archive gives out a folder named `foobar/`) the archive and set it in `builddir="${srcdir}/foobar/"`.

Is that it?

(Maybe I'm being picky on those questions, but I'm unsure about the requests, they seem arbitrary and want to understand. We could document those, if they're frequent)